client/settings/fraud-protection/index.js

```diff
@@ -0,0 +1,501 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { sprintf, __ } from '@wordpress/i18n';
+import { Button, Card, Modal, Notice } from '@wordpress/components';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import CardBody from '../card-body';
+import { useCurrentProtectionLevel, useCurrencies } from '../../data';
+import TipIcon from 'wcpay/icons';
+import './style.scss';
+
+const ExceedsDollarAmountRuleUSDStore = ( props ) => {
+	if ( 'high' === props.level ) {
+		return (
+			<li>
+				{ __( 'An order exceeds ', 'woocommerce-payments' ) }{ ' ' }
+				<strong>{ __( '$1,000.00.', 'woocommerce-payments' ) }</strong>
+			</li>
+		);
+	}
+
+	return (
```

Comment:
As both paths share the same initial component structure, it could be rewritten to something like this:

```js
const ExceedsDollarAmountRuleUSDStore = ( props ) => {
	const isHighProtectionLevel = 'high' === props.level;

	return (
		<li>
			{ __( 'An order exceeds', 'woocommerce-payments' ) }{ ' ' }
			<strong>{ __( '$1,000.00', 'woocommerce-payments' ) } </strong>

			{ ! isHighProtectionLevel && (
				<>
					{ __( 'or', 'woocommerce-payments' ) }{ ' ' }
					<strong>
						{ __( '10 items.', 'woocommerce-payments' ) }
					</strong>
				</>
			) }
		</li>
	);
};
```

Reply:
I'm not sure this would work for the USD store component as the high level text has a period after the dollar amount: `'$1,000.00.'`